qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -290,3 +310,40 @@ def test_listing_of_ui_api_ls_on_deeper_levels(self):
         ui_api_ls = self.ui_ls_dir('/pictures', 0)
         self.assertEqual(api_ls, ui_api_ls)
         self.rm_dir('/pictures')
+
+    def test_statfs(self):
+        self.statfs('/')
+
+        self.mk_dirs('/animal')
+        date = self.statfs('/animal')
```

Comment:
`date`? is it a typo? should be `data` or better would be `stats` since the name of the test case itself is for stats

Reply:
sorry, it is a typo.